Makefile

```diff
@@ -0,0 +1,17 @@
+box.phar:
```

Comment:
you probably want to add `touch $@` at the end of each to ensure the file timestamp is properly updated